### PR TITLE
(bugsbash)Package "prometheus/prometheus/pkg/labels" imported twice

### PR DESCRIPTION
**Pull request by asquare14**
Signed-off-by: aSquare14 <atibhi.a@gmail.com>

* [ ] I added CHANGELOG entry for this change.
* [x ] Change is not relevant to the end user.

## Changes
 Fixes ST1019 : package "github.com/prometheus/prometheus/pkg/labels" is being imported more than once

### CODE REVIEW

**Comment by squat** [comment]
Thank you @asquare14! Could you please sign the DCO (hint git commit --amend --signoff) and push again?

**Comment by asquare14** [comment]
Ahh there's an issue with my laptop and by mistake I removed all the CLI tools. Waiting for some software to get installed. Will signoff in an hour or so. Meanwhile looking at how to signoff using GitHub UI. Thanks for the review !

**Comment by asquare14** [comment]
@squat Done !

File: pkg/promclient/promclient.go [hunk 1]
```diff
@@ -26,7 +26,6 @@ import (
 	"github.com/gogo/status"
 	"github.com/pkg/errors"
 	"github.com/prometheus/common/model"
-	"github.com/prometheus/prometheus/pkg/labels"
```
Comment (squat) [inline pkg/promclient/promclient.go:29]: Ok, so it looks like we are using both of the names for this package in different places. Some lines, e.g. line 160/159 use the `labels` name for this package whereas other places refer to it as `promlabels`. To make this change work, we need to rename all of the usages to make sure that they all use either `labels` or `promlabels` but not both

**Review by squat — COMMENTED** [review]
Thanks for following up :) just need to do one more thing to make the code compile

File: pkg/promclient/promclient.go (hunk 1 above)
Comment (asquare14) [reply pkg/promclient/promclient.go:29]: Done ! Changed promlabels to labels for consistency.

**Review by squat — APPROVED** [review]
Amazing, thank you!

**Pull request merged**